src/test/erc4626/ERC4626.t.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+
+import {Vm} from '../../../lib/forge-std/src/Vm.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {DefiBridgeProxy} from './../../aztec/DefiBridgeProxy.sol';
+import {RollupProcessor} from './../../aztec/RollupProcessor.sol';
+import {IERC4626} from './../../bridges/erc4626/Interfaces/IERC4626.sol';
+import {VaultBridge} from './../../bridges/erc4626/VaultBridge.sol';
+import {AztecTypes} from './../../aztec/AztecTypes.sol';
+import {console} from '../console.sol';
+import '../../../lib/ds-test/src/test.sol';
+
+//Tested at block 14886873 may not work at other blocks
+contract ERC4626 is DSTest {
```

Comment:
Please extend Test class from forge-std library instead of importing DSTest directly. What is an advantage of using the class from forge-std is that there is a deal functionality which can be used for token minting. The advantage of deal(...) over the _setTokenBalance(...) is that it automatically detects the slot of mapping with balances.

Please check out [this example](https://github.com/AztecProtocol/aztec-connect-bridges/blob/e53c3f2fe746aa02f680571f29c08f02a90d932d/src/test/example/Example.t.sol#L14).